WordPress/gutenberg
# Build Tooling: Remove WebpackRTLPlugin

Closes #15146 

This pull request seeks to remove the `WebpackRTLPlugin`. The plugin is no longer necessary, as there is otherwise RTL CSS generated through the packages build script. The Webpack build configuration is only responsible for copying these built files:

https://github.com/WordPress/gutenberg/blob/d08dcd89fd83cdcd0f899b3df8c4a0d4565976bd/webpack.config.js#L85-L107

The plugin is also slated to become broken due to removed deprecations with Webpack 5.x , and would otherwise need to be updated.

**Testing Instructions:**

Verify there are no regressions in the CSS RTL output from the build.

cc @yoavf
